src/utils.ts

```diff
@@ -24,6 +24,9 @@ export function getPipePath(pipeName: string) {
 // Check that the file or directory exists in an asynchronous manner that relies
 // solely on the VS Code API, not Node's fs library, ignoring symlinks.
 async function checkIfFileOrDirectoryExists(targetPath: string | vscode.Uri, type: vscode.FileType): Promise<boolean> {
+    if (targetPath === "") {
```

Comment:
@SeeminglyScience since the settings overhaul PR, the default `cwd` was `""` which conspicuously `vscode.workspace.fs.stat` was _totally fine_ saying it existed.